templates/vt_device_driver_template.h

```diff
@@ -0,0 +1,131 @@
+/* Copyright (c) Microsoft Corporation.
+   Licensed under the MIT License. */
```

Comment:
Is this file something that would be modified by the customer? If not, I think this should live in the "core/platform" and remove the template.
All that should be needed is for someone to copy the template.c file into their project, implement the functions and link it in.

Reply:
fixed